test/Module/CreateTest.php

```diff
@@ -129,7 +129,7 @@ public function testCreatesConfigProvider()
         $this->assertSame(1, preg_match('/\bnamespace MyApp\b/', $configProviderContent));
         $this->assertSame(1, preg_match('/\bclass ConfigProvider\b/', $configProviderContent));
         $command = $this->command;
-        $expectedContent = sprintf($command::TEMPLATE_CONFIG_PROVIDER, 'MyApp', 'myapp');
+        $expectedContent = sprintf($command::TEMPLATE_CONFIG_PROVIDER, 'MyApp', 'my-app');
```

Comment:
I am not sure if that test case covers all changes in `createTemplateNamespace`. Maybe we need to add something more (we have there two regular expressions?).

Reply:
This test is just testing if the ConfigProvider is the right format. There could be more tests for template namespace.

Reply:
I'd say that once you extract the trait, you should add the additional tests as well.